Source/RageCommunity.Library/Extensions/PedExtensions.cs

```diff
@@ -40,5 +40,12 @@ public static List<PedTask> GetAllActiveTasks(this Ped ped)
             var tasks = (PedTask[])Enum.GetValues(typeof(PedTask));
             return tasks.Where(t => ped.IsTaskActive(t)).ToList();
         }
+        /// <summary>
+        /// indicates whether this <paramref name="ped"/> can see the <paramref name="target"/> <see cref="Ped"/>
+        /// </summary>
+        public static bool CanSee(this Ped ped, Ped target)
```

Comment:
I recommend adding Ped to the name of the method for more clarity, otherwise someone might believe it can check to see if the ped can see any entity instead of just another ped.
```suggestion
        public static bool CanSeePed(this Ped ped, Ped target)
```